tests/TestGraph.hs

```diff
@@ -4,11 +4,8 @@ import Prelude ()
 import Prelude.Compat
 
 import Test.Hspec
-import System.IO.UTF8 (readUTF8FileT)
 import Data.Either (isLeft)
 
-import qualified Data.ByteString.Lazy as ByteString
-import qualified Data.Text.Encoding as Text
 import qualified Data.Aeson as Json
```

Comment:
I think these imports would have to be restored for this to build properly.
```suggestion
import System.IO.UTF8 (readUTF8FileT)
import Data.Either (isLeft)

import qualified Data.Aeson as Json
import qualified Data.ByteString.Lazy as ByteString
import qualified Data.Text.Encoding as Text
```

Reply:
Ah, it looks like e97495c263cbe96c6fc54603b9d6419612135d72 introduced a conflict with this PR. It should work without all the extra encoding/decoding.